tool/jt.rb

```diff
@@ -833,7 +833,11 @@ def test_mri(*args)
   private :test_mri
 
   def run_mri_tests(extra_args, test_files, run_options = {})
-    truffle_args = %w[-J-Xmx2G -J-ea -J-esa --jexceptions]
+    truffle_args =  if extra_args.include?('--aot')
+                      %W[-XX:YoungGenerationSize=2G -XX:OldGenerationSize=4G -Xhome=#{TRUFFLERUBY_DIR}]
```

Comment:
Is there a way to enable assertions on AOT? That would pretty useful to debug failing tests.
But I guess that needs to be cared for at image building time.